WordPress/Classes/ViewRelated/Stats/Charts/PeriodChart.swift

```diff
@@ -114,23 +114,21 @@ private final class PeriodChartDataTransformer {
 
         var chartData = [BarChartData]()
 
-        let viewsDataSet = BarChartDataSet(values: viewEntries,
-                                           label: NSLocalizedString("Views", comment: "Accessibility label used for distinguishing Views and Visitors in the Stats → Views bar chart."),
-                                           valueFormatter: dataSetValueFormatter)
-        let visitorsDataSet = BarChartDataSet(values: visitorEntries,
-                                              label: NSLocalizedString("Visitors", comment: "Accessibility label used for distinguishing Views and Visitors in the Stats → Views bar chart."),
-                                              valueFormatter: dataSetValueFormatter)
+        let viewsDataSet = BarChartDataSet(entries: viewEntries,
+                                           label: NSLocalizedString("Views", comment: "Accessibility label used for distinguishing Views and Visitors in the Stats → Views bar chart."))
+        let visitorsDataSet = BarChartDataSet(entries: visitorEntries,
+                                              label: NSLocalizedString("Visitors", comment: "Accessibility label used for distinguishing Views and Visitors in the Stats → Views bar chart."))
         let viewsDataSets = [ viewsDataSet, visitorsDataSet ]
         let viewsChartData = BarChartData(dataSets: viewsDataSets)
         chartData.append(viewsChartData)
 
         let visitorsChartData = BarChartData(dataSet: visitorsDataSet)
         chartData.append(visitorsChartData)
 
-        let likesChartData = BarChartData(entries: likeEntries, valueFormatter: dataSetValueFormatter)
+        let likesChartData = BarChartData(entries: likeEntries)
         chartData.append(likesChartData)
 
-        let commentsChartData = BarChartData(entries: commentEntries, valueFormatter: dataSetValueFormatter)
+        let commentsChartData = BarChartData(entries: commentEntries)
```

Comment:
⚠️ I've gone back and forth on these and I still need to finish upgrading. The current version would introduce a regression.